backend/src/main/java/com/redhat/cloud/notifications/db/Query.java

```diff
@@ -1,12 +1,15 @@
 package com.redhat.cloud.notifications.db;
 
 import io.quarkus.logging.Log;
+import io.sentry.Sentry;
```

Comment:
```suggestion
```

Reply:
Always forget to clean my imports :sweat_smile: